de-webapp/src/main/resources/templates/hello.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Hello Spring Boot!</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+</head>
+<body>
+<p>Hello Spring Boot!</p>
+</body>
+</html>
```

Comment:
The newer version of Spring Boot complained about the missing directory, so I just put a fake file in here.